niri-session is a wrapper over systemctl/dinitctl to start a user service;
it doesn't have a proper fallback ( instead, just prints a message to stdout ),
so instead we can just launch niri directly to guarantee display managers can launch it

diff --git a/resources/niri.desktop b/resources/niri.desktop
index 5470fe7..0164374 100644
--- a/resources/niri.desktop
+++ b/resources/niri.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=Niri
 Comment=A scrollable-tiling Wayland compositor
-Exec=niri-session
+Exec=niri --session
 Type=Application
 DesktopNames=niri
